Ginger/GingerTest/Variable Dependancies/VariableDependancyTest.cs

```diff
@@ -0,0 +1,220 @@
+﻿
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.InterfacesLib;
+using Amdocs.Ginger.Repository;
+using Ginger.Run;
+using GingerCore;
+using GingerCore.Actions;
+using GingerCore.Activities;
+using GingerCore.Environments;
+using GingerCore.Platforms;
+using GingerCore.Variables;
+using GingerCoreNET.SolutionRepositoryLib.RepositoryObjectsLib.PlatformsLib;
+using GingerTestHelper;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace GingerTest.Variable_Dependancies
+{
+    [TestClass]
+    [Level1]
+    public class VariableDependancyTest
+    {
+        static BusinessFlow BF1;
+        static GingerRunner mGR;
+        static ProjEnvironment environment;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field 'environment' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4817688)